.vsts/darwin/notarize.yml

```diff
@@ -11,7 +11,7 @@ steps:
             "KeyCode" : "CP-401337-Apple",
             "OperationCode" : "MacAppNotarize",
             "Parameters" : {
-              "BundleId": "com.microsoft.azure.BatchExplorer"
+              "BundleId": "com.microsoft.BatchExplorer"
```

Comment:
Is this the correct bundle ID? If so I think we need to update electron-builder.yml as well.